Police-Data-Accessibility-Project/data-sources-app
# Feature/update design system

# Bump `design-system` version, implement component library

- [x] Bump `design-system` -> `v2.0.1`
- [x] Use components from `design-system@2.0.1`
- [x] Remove app-specific components
Resolves #85 

## To test
Run flask app and client locally, ensure styles match PDAP branding and that everything functions as expected.
